lib/faktory/job.rb

```diff
@@ -29,6 +29,27 @@ def self.included(base)
       base.faktory_class_attribute :faktory_options_hash
     end
 
+    def self.set(options)
```

Comment:
I'd like to see `set` and `client_push` as instance methods. Class methods are a definite smell. Perhaps this code could move into Faktory::Client, just like Sidekiq::Client.

Reply:
Mmm, I see.  `Foo.set(queue: 'bar').perform_async(...)` must be a thing so belay that for `set`...

Reply:
I moved `client_push` to be an instance method on `Setter` for now.
I've tried moving to `Client` but weird things started to happen, b/c everything kind of assumes `Client#push` already gets a normalized job Hash.